protocol/writing-resource/delete-protect-nonempty-container.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Server protects non-empty container
+  
+  Background: Set up container and a few child resources
+    * def testContainer = rootTestContainer.createContainer()
+  
+  Scenario: Check that container with child container is protected
+    * def container1 = testContainer.createContainer()
```

Comment:
```suggestion
    * testContainer.createContainer()
```
Since you don't need `container1` this can be simplified.
See: https://github.com/karatelabs/karate#eval